evm/src/cpu/cpu_stark.rs

```diff
@@ -125,38 +125,12 @@ pub fn ctl_arithmetic_shift_rows<F: Field>() -> TableWithColumns<F> {
     TableWithColumns::new(Table::Cpu, columns, Some(Column::single(COL_MAP.op.shift)))
 }
 
-pub fn ctl_data_byte_packing<F: Field>() -> Vec<Column<F>> {
+pub fn ctl_data_byte32<F: Field>() -> Vec<Column<F>> {
     ctl_data_keccak_sponge()
 }
```

Comment:
nit: as we're re-using `ctl_data_keccak_sponge` both for `MLOAD_32BYTES` and `MSTORE_32BYTES`, perhaps a tiny comment explaining the distinction may be useful? (as the doc there is valid for MLOAD but the last memory channel is being read in the case of MSTORE, not pushed to).